src/main/java/org/jenkinsci/plugins/github/util/JobInfoHelpers.java

```diff
@@ -111,7 +113,8 @@ public boolean apply(ITEM item) {
         if (item instanceof ParameterizedJobMixIn.ParameterizedJob) {
             ParameterizedJobMixIn.ParameterizedJob pJob = (ParameterizedJobMixIn.ParameterizedJob) item;
 
-            for (Trigger candidate : pJob.getTriggers().values()) {
+            Map<TriggerDescriptor, Trigger<?>> triggerMap = pJob.getTriggers();
```

Comment:
can you give more details regarding this change?

Reply:
return type (generics) is changed on new core